haskell-style.md

```diff
@@ -872,6 +872,25 @@ Good
 -- | Returns the head of a non-empty list, or raises an exception
 ```
 
+When a Body is _not_ present (see below), no newline is required between a Summary and its associated top-level definition:
```

Comment:
Do you mind hard-wrapping this at 80?

We don't have a Markdown style guide yet, but I'm working on it :)